toolchain/check/check_unit.cpp

```diff
@@ -335,34 +335,50 @@ auto CheckUnit::ImportOtherPackages(SemIR::TypeId namespace_type_id) -> void {
 }
 
 auto CheckUnit::ImportCppPackages() -> void {
-  for (const auto& import : unit_and_imports_->cpp_imports) {
-    llvm::StringRef cpp_file_path =
-        unit_and_imports_->unit->value_stores->string_literal_values().Get(
-            import.library_id);
-
-    auto file = fs_->openFileForRead(cpp_file_path);
-    if (!file) {
-      CARBON_DIAGNOSTIC(CppInteropFileNotFound, Error,
-                        "file '{0}' couldn't be opened for reading: {1}",
-                        std::string, std::string);
-      emitter_.Emit(import.node_id, CppInteropFileNotFound, cpp_file_path.str(),
-                    file.getError().message());
-      continue;
-    }
+  const auto& imports = unit_and_imports_->cpp_imports;
+  if (imports.empty()) {
+    return;
+  }
 
-    llvm::vfs::File& file_ref = *file.get();
-    auto buffer = file_ref.getBuffer(cpp_file_path);
-    if (!buffer) {
-      CARBON_DIAGNOSTIC(CppInteropFailedAccessingFileBuffer, Error,
-                        "failed accessing buffer of file '{0}': {1}",
-                        std::string, std::string);
-      emitter_.Emit(import.node_id, CppInteropFailedAccessingFileBuffer,
-                    cpp_file_path.str(), buffer.getError().message());
-      continue;
-    }
-    ImportCppFile(context_, import.node_id, cpp_file_path,
-                  buffer.get()->getBuffer());
+  if (imports.size() >= 2) {
+    // TODO: Support multiple Cpp imports.
+    CARBON_DIAGNOSTIC(CppInteropMultipleImports, Error,
+                      "multiple ({0}) Cpp imports are not supported", int);
```

Comment:
```suggestion
                      "multiple ({0}) Cpp imports are not yet supported", int);
```

Reply:
Had you considered using `context.TODO` for this, instead of defining a diagnostic?